js/buttons.html5.js

```diff
@@ -865,19 +865,20 @@ DataTable.ext.buttons.copyHtml5 = {
 			textarea[0].select();
 
 			try {
-				document.execCommand( 'copy' );
+				var successful = document.execCommand( 'copy' );
 				hiddenDiv.remove();
```

Comment:
It doesn't really make any difference. It is appended to the manual copy div if the code gets that far, so it doesn't really matter if the element is in the DOM or not at that point. I'm happy with it as it is.